include/crypto/aes_platform.h

```diff
@@ -158,6 +158,13 @@ void gcm_ghash_v8(u64 Xi[2],const u128 Htable[16],const u8 *inp, size_t len);
 #  define AESNI_CBC_HMAC_SHA_CAPABLE (OPENSSL_ia32cap_P[1]&(1<<(57-32)))
 # endif
 
+#if defined(__loongarch__) || defined(__loongarch64)
+#include "loongarch_arch.h"
+# if defined(VPAES_ASM)
+#   define VPAES_CAPABLE  (OPENSSL_loongarchcap_P & LOONGARCH_CFG2_LSX)
+# endif
+#endif
```

Comment:
Please make the preprocessor directive indentations right. I.e., add one more space in front of everything, except the `#   define`, and add two spaces in `#include'.

Reply:
Thank you very much for your review. preprocessor directive indentations has been corrected.